src/main/java/cam72cam/immersiverailroading/library/ChatText.java

```diff
@@ -29,6 +29,9 @@ public enum ChatText {
 	RADIO_NOLINK("radio.nolink"),
 	RADIO_CANT_LINK("radio.cant_link"),
 	BRUSH_NO_VARIANTS("brush.no.variants"),
+	BRUSH_NEXT("brush.next"),
+	BRUSH_MODE_SET("brush.mode.set"),
+	BRUSH_MODE_NAME_PREFIX("brush.mode."),
```

Comment:
This should be a constant string in the other enum.  This one assumes it's a fully qualified translation.